providers/terraform-provider-csbmssqldbrunfailover/csbmssqldbrunfailover/config.go

```diff
@@ -7,9 +7,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+const (
+	maxLengthClientSecret = 1024
+)
+
 var (
-	identifierRegexp   = regexp.MustCompile(`^[\w_.-]{1,64}$`)
-	clientSecretRegexp = regexp.MustCompile(`^[\w_.-]{1,96}$`)
+	identifierRegexp = regexp.MustCompile(`^[\w_.-]{1,64}$`)
```

Comment:
We were wondering whether this regular expression might cause problems in the future if there's a valid identifier that doesn't meet our expectations. Should we relax this check too?